iaso/api/completeness_stats.py

```diff
@@ -226,16 +227,21 @@ def list(self, request: Request, *args, **kwargs) -> Response:
         orders = params["order"]
         form_qs = params["forms"]
         period = params.get("period", None)
-        planning = params.get("planning", None)
+        planning_id = params.get("planning_id", None)
         org_unit_validation_status = params["org_unit_validation_status"]
+        as_location = params.get("as_location", None)
 
         instance_qs = Instance.objects.all()
+
+        planning = None
+        if planning_id:
+            planning = get_object_or_404(Planning, id=planning_id)
```

Comment:
As I don't have all the context I'm not sure if we really want to return a 404 at this line. Maybe a queryset would be better ? At line 243 and 302 there is a check for planning so I guess it can still run without a planning. An empty queryset is evaluated as None so it would not break anything I guess